cpp/src/arrow/adapters/arrow-rados-cls/README.md

```diff
@@ -76,3 +76,31 @@ docker-compose run --service-ports ubuntu-cls-demo
 # Development
 
 Check out these [instructions](docs/contributing.md) for setting up a local development environment.
+
+## Set up the development environment using docker
+
+1. Clone the SkyhookDM repository.
+```bash
+git clone https://github.com/uccross/skyhookdm-arrow
+cd skyhookdm-arrow/
+```
+
+2. Run the SkyhookDM container in interactive mode.
+```bash
+docker run -it -v $PWD:/w -w /w --privileged uccross/skyhookdm-arrow:v0.2.0 bash
+```
+
+3. Install docker.
+```bash
+curl -o- https://get.docker.com | bash
+```
+
+3. Run the build script.
+```bash
+./build.sh
+```
+
+4. Run the tests script.
+```bash
+./test.sh
```

Comment:
Same for this

Reply:
This was in the README instructions that was mentioned in the issue. Is there some other way I should import them as was said in the issue?

Reply:
Yes, bring the `build.sh` and `test.sh` scripts to this repo also.

Reply:
Okay. Done.